reconcile/utils/jjb_client.py

```diff
@@ -238,13 +238,13 @@ def update(self):
                 result = subprocess.run(cmd,
                                         check=True,
                                         stdout=PIPE,
-                                        stderr=STDOUT)
-                out_str = result.stdout.decode("utf-8")
-                if re.search("updated: [1-9]", out_str):
-                    logging.info(out_str)
+                                        stderr=STDOUT,
+                                        encoding="utf-8")
+                if re.search("updated: [1-9]", result.stdout):
+                    logging.info(result.stdout)
             except CalledProcessError as ex:
-                msg = ex.stdout.decode("utf-8")
-                logging.error(msg)
+                logging.error(ex.stdout)
```

Comment:
Why bother? Why not just remove the `try` block here and let somebody up the stack do the logging?

Reply:
CalledProcessError stdout attribute contains the output of the failed command and this data is not printed with the base exception handling. I think it's worth to log the command's output and not just `command returned with non-zero error`. The exception is raised and managed up in the stack though.

Reply:
Fair enough. I'm convinced.